hashtables/ex1/ex1.py

```diff
@@ -12,10 +8,19 @@ def get_indices_of_item_weights(weights, length, limit):
     """
     YOUR CODE HERE
     """
+    for i in range(length):
+        hash_table_insert(ht, weights[i], i)
+
+    for i in range(length):
```

Comment:
consider adding the functionality of this for loop to the first one.  that is, we can check the hash table for complimenting values while we are building in, instead of waiting until after it is fully build.